### ISSUE TITLE
passthru requires guest memory to be wired

### ISSUE THREAD

**Issue by WanpengQian** [churchers/vm-bhyve#78]
FreeBSD 10.3R
vm: Bhyve virtual machine management v0.10.1

after adding 
```
passthru0="14/0/0"
```
root@x220:/zones/vm/vm168 # cat bhyve.log 
passthru requires guest memory to be wired

vm-bhyve.log
```
Apr 05 22:49:34: booting
Apr 05 22:49:34:  [bhyve options: -c 2 -m 2G -Hwl bootrom,/zones/vm/.config/BHYVE_UEFI.fd -U 527190c4-7775-11e5-8831-0021cc60a290]
Apr 05 22:49:34:  [bhyve devices: -s 0,hostbridge -s 31,lpc -s 4:0,ahci-hd,/dev/zvol/zones/vm/vm168/disk0 -s 5:0,virtio-net,tap0,mac=00:A0:98:20:45:1B -s 6:0,passthru,14/0/0]
Apr 05 22:49:34:  [bhyve console: -l com1,/dev/nmdm0A]
Apr 05 22:49:34:  [bhyve iso device: -s 3:0,ahci-cd,/zones/vm/.config/null.iso]
Apr 05 22:49:34: starting bhyve
Apr 05 22:49:34: bhyve exited with status 1
Apr 05 22:49:34: destroying network device tap0
Apr 05 22:49:34: stopped
```
it seems not adding -S to bhyve command line?

**Comment by WanpengQian** [comment]
in vm-run
```
        # add wired memory for 11+
        [ ${BSD_VERSION} -ge 1100000 ] && _opts="${_opts} -S" && _wiredmem="1"
```
$BSD_VERSION check is incorrect.

**Comment by churchers** [comment]
The wiki says only 11+ requires wired memory :(

    Note that post r284539 the "-S" option must be passed to both bhyveload and bhyve to wire guest memory. Also note that this is required only in FreeBSD 11.0 and above.

They must of merged the changes back into 10-stable before 10.3 was released.
I'll update the check
